lib/path.js

```diff
@@ -437,15 +437,26 @@ const win32 = {
     if (from === to)
       return '';
 
-    const fromOrig = win32.resolve(from);
-    const toOrig = win32.resolve(to);
+    let fromOrig = win32.resolve(from);
+    let toOrig = win32.resolve(to);
 
     if (fromOrig === toOrig)
       return '';
 
     from = fromOrig.toLowerCase();
     to = toOrig.toLowerCase();
 
+    if (fromOrig.length !== from.length) {
+      fromOrig = fromOrig.normalize('NFD');
+      from = fromOrig.toLowerCase();
+    }
+    let toNormalized;
```

Comment:
⚙️ (optional) Let's use `false` rather than `undefined` since it's boolean type:
```suggestion
    let toNormalized = false;
```